providers/shared/references/Port/id.ftl

```diff
@@ -23,7 +23,18 @@
         },
         {
             "Names" : "Port",
-            "Type" : NUMBER_TYPE
+            "Children" : [
```

Comment:
This is replaced by my suggestion above

Reply:
Reference data is shared data between all component types. And not all component types care about Http or Https

Reply:
Have completed this in dbf1030